### PR TITLE
add api diff between preview1 & preview2

### PR DESCRIPTION
**Pull request by Anipik**
@buyaa-n @joperezr @krwq @eiriktsarpalis @layomia @tommcdon @Anipik @ViktorHofer @dotnet/ncl @pgovind @tannergooding @steveharter

@ericstj

@pranavkm @Tratcher @Pilchie @davidfowl @dougbu @richlander

### CODE REVIEW

**Review by ericstj — APPROVED** [review]
I skimmed libraries changes and it LGTM

**Review by eiriktsarpalis — APPROVED** [review]
LGTM from the perspective of Collections & LINQ

**Review by buyaa-n — APPROVED** [review]
CodeDom.Compiler IndentedTextWriter changes LGTM

File: release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Numerics.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
```
Comment (tannergooding) [inline release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Numerics.md:4]: System.Numerics changes LGTM.

CC. @pgovind

File: release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Runtime.Intrinsics.Arm.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# System.Runtime.Intrinsics.Arm
+
+``` diff
+ namespace System.Runtime.Intrinsics.Arm {
```
Comment (tannergooding) [inline release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Runtime.Intrinsics.Arm.md:4]: System.Runtime.Intrinsics changes LGTM.

CC. @pgovind, @echesakovMSFT

File: release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# System
+
+``` diff
+ namespace System {
+     public readonly struct IntPtr : IComparable, IComparable<IntPtr>, IEquatable<IntPtr>, IFormattable, ISerializable {
```
Comment (tannergooding) [inline release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.md:5]: `System.IntPtr` changes LGTM.

CC. @pgovind

**Review by tannergooding — APPROVED** [review]

File: release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Runtime.Intrinsics.Arm.md (hunk 2 above)
Comment (echesakovMSFT) [reply release-notes/6.0/preview/api-diff/preview2/.Net/6.0-preview2_System.Runtime.Intrinsics.Arm.md:4]: Reviewed `System.Runtime.Intrinsics.Arm`

LGTM

**Review by echesakovMSFT — APPROVED** [review]
Reviewed `System.Runtime.Intrinsics.Arm`

LGTM

**Pull request merged**